AWSPinpoint/AWSPinpointEndpointProfile.m

```diff
@@ -546,4 +561,83 @@ - (void)encodeWithCoder:(NSCoder *)encoder {
     [encoder encodeObject:_userId forKey:@"userId"];
```

Comment:
Now that we've added userAttributes to this class, seems like we would need to add it to this function `encodeWithCoder:` ?